### PR TITLE
add mediborg to silicons department

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
title

## Why / Balance
borg is silicon
mediborg is silicon

no cl

### CODE REVIEW

**Comment by ghost** [comment]
`The job MedicalBorg has more than 1 primary department!`

`Assert.That(primaries, Is.EqualTo(1))`
  `Expected: 1`
  `But was:  2`
  
  You should probably do this for any other borgs that have job prototypes / things in anticipation for AI (I say that because Secborg might have one.)

File: Resources/Prototypes/Roles/Jobs/departments.yml
Comment (MilonPL) [inline Resources/Prototypes/Roles/Jobs/departments.yml:1]: You have to change silicon to not be a primary department for the tests to pass. Having no primary department (like captain does) is fine and shouldn't break anything.

**Review by MilonPL — CHANGES_REQUESTED** [review]

**Comment by TadJohnson00** [comment]
@deltanedas thoughts on putting station AI under the command dept. as well?

**Comment by deltanedas** [comment]
it makes sense playtime wise and i think its how a lot of ss13 servers do it too

**Review by TadJohnson00 — APPROVED** [review]
dir. approved or some shit

**Pull request merged**